core/src/main/resources/reference.conf

```diff
@@ -56,7 +56,7 @@ akka {
 
 system {
   # If 'true', a SIGINT will trigger Cromwell to attempt to abort all currently running jobs before exiting
-  abort-jobs-on-terminate = false
+  #abort-jobs-on-terminate = false
```

Comment:
Leaving, it's not comment text, but a commented-out value. The lack of space matches other commented-out values in the file.